server/auth/index.ts

```diff
@@ -94,11 +100,11 @@ export async function requireOrgAccess(req: any, res: Response, next: NextFuncti
   }
   const user: User = req.appUser;
   if (orgId && user) {
-    const orgRole = user.roles!.find(role => role.org!.id === orgId);
-    if (orgRole) {
-      req.appOrg = orgRole.org;
-      req.appRole = orgRole;
-      req.appWorkspace = orgRole.workspace;
+    const orgUserRole = user.userRoles.find(userRole => userRole.role.org!.id === orgId);
+    if (orgUserRole) {
+      req.appOrg = orgUserRole.role.org;
+      req.appRole = orgUserRole.role;
```

Comment:
Here's the core part of what I think will need to be refactored to switch the entire backend over to using `UserRole` instead of `Role`. This should be changed to `req.appUserRole` with its type changed to `UserRole`.